Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM sonarsource/sonar-scanner-cli:5.0
+FROM sonarsource/sonar-scanner-cli:5.0.0
```

Comment:
Suggestion: maybe pin it to the [hash](https://hub.docker.com/layers/sonarsource/sonar-scanner-cli/5.0.0/images/sha256-2240c1507a7b9c0ce5a8ffb9bfb94a85ec321f2e6527ced16c31dd727e099cf2?context=explore) instead of the tag?

```suggestion
FROM sonarsource/sonar-scanner-cli@sha256:2240c1507a7b9c0ce5a8ffb9bfb94a85ec321f2e6527ced16c31dd727e099cf2
```

Reply:
The versions apparently published are:

![image](https://github.com/SonarSource/sonarcloud-github-action/assets/84329129/fd91a797-07e6-4748-9598-c87e0750af03)


So I dont know if that's going to work (the original PR)